scipy/stats/stats.py

```diff
@@ -2789,10 +2789,17 @@ def median_absolute_deviation(x, axis=0, center=np.median, scale=1.4826,
         A function that will return the central value. The default is to use
         np.median. Any user defined function used will need to have the function
         signature ``func(arr, axis)``.
-    scale : int, optional
-        The scaling factor applied to the MAD. The default scale (1.4826)
-        ensures consistency with the standard deviation for normally distributed
-        data.
+    scale : scalar or str, optional
+        The numerical value of scale will be divided out of the final
+        result. The following string values are recognized (default is 'raw'):
+
+          * 'raw' : No scaling, just return the raw sum of absolute deviations.
+          * 'normal' : Scale by :math:`\frac{1}{\sqrt{2} erf^{-1}(\frac{1}{2})}\approx 0.67449`.
```

Comment:
These didn't have `*` and weren't indented correctly and we causing doctests to fail.